GP made Optional Patient PDS

NaminderSoorma opened the pull request:
GP made Optional Patient PDS

NaminderSoorma committed:
GP made Optional Patient PDS

GP made Optional Patient PDS

omarisgreat commented on StructureDefinition/NHSDigital-Patient-PDS.StructureDefinition.xml:
```diff
@@ -38,28 +46,13 @@
-      <max value="1" />
+    <element id="Patient.address.extension:addressKey.extension">
+      <path value="Patient.address.extension.extension" />
+      <min value="2" />
```
Could we remove the constraint on addressKey entirely? There may be up to 2 for PDS there may also be none at all.

omarisgreat reviewed (changes requested):
1 change requested, fine with the rest

NaminderSoorma committed:
EPS changes to remove items

EPS changes to addresskey to make it same as UKCore-Patient, also align the Patient and PatientPDS to be same on these elements.

NaminderSoorma reviewed (commented):
Changes made to include EPS requirements for Patient addresskey.

omarisgreat reviewed (approved):
Looks good to me
